### DISCUSSION TITLE
Easier way to tell where a wire is going

### DISCUSSION THREAD

**Discussion by GoodDays13** [FakeFishGames/Barotrauma#10532] [Ideas and suggestions]
It is a bit hard to know where a wire is going when they go off-screen.
![image](https://media.discordapp.net/attachments/220881718666723328/1045144757971734528/image.png)
I wish some electric components had IDs or numbers, and they would show next to the wire in wiring interfaces, so I know what wire is connected to what.
Such as like this:
![image](https://user-images.githubusercontent.com/42877449/203848897-3da67211-1ee9-4d67-9abb-42c0a31c11e9.png)
And maybe the wiring interface could have the name and number in the top
![image](https://media.discordapp.net/attachments/220881718666723328/1045144218886226000/image.png)

An alternative could be a way to keep the wires highlighted so you could follow it. It could be an engineer trait because they can follow the wires from experience maybe.

**Comment by gammy** [comment]
I've been thinking of something along the same lines, but it is perhaps difficult to implement; I don't know:

An input/output of an electrical component could have an arbitrary text attributed to it. 
Imagine an engineer of level/perk X changing the name of `SIGNAL_0` to `Hatch input`. This would add some realism, as well as improved game mechanics: an engineer on a new submarine would have better understanding of how the submarine is wired, and they could also mess up the labels (either by mistake or on purpose). 

This approach could perhaps render components (especially submarine built-ins) unusable if labeling mistakes were made, but maybe one could reset them, or just have such labels as an addendum to the built-in name : `SIGNAL_0` becomes `SIGNAL_0 (_arbitrary text here_)`.
